### PR TITLE
Passes attr to update_user_with_attributes_sql

### PR DESCRIPTION
**Pull request by Zhann**
# Description

Passes `attr` to `update_user_with_attributes_sql` to make updates on `postgresql_user` work with attribute hashes.

Also removes quotes from rolname, as that's a syntax error in postgresql.

## Issues Resolved

Fixes https://github.com/sous-chefs/postgresql/issues/653

## Check List

- [ ] All tests pass. See TESTING.md for details.
- [x] New functionality includes testing.
- [ ] ~~New functionality has been documented in the README if applicable~~.

### CODE REVIEW

**Comment by damacus** [comment]
Hey there!

Thanks for the PR. It might be worth putting a test in to exercise this. 

Cheers,
Dan

**Comment by Zhann** [comment]
Thanks @damacus, added an integration test. Moved it to the access integration tests, not sure if it makes sense to put it there, but it felt like the closest fit. Let me know if you think I should do it differently.

File: libraries/helpers.rb [hunk 1]
```diff
@@ -127,8 +127,8 @@ def update_user_sql(new_resource)
       psql_command_string(new_resource, sql)
     end
 
-    def update_user_with_attributes_sql(new_resource, value)
-      sql = %(ALTER ROLE '#{new_resource.create_user}' SET #{attr} = #{value})
+    def update_user_with_attributes_sql(new_resource, attr, value)
+      sql = %(ALTER ROLE #{new_resource.create_user} SET #{attr} = #{value})
```
Comment (Xorima) [inline libraries/helpers.rb:131]: Question about this change, (not a postgresql user) is it possible to create a user with a space in the name?

**Comment by Xorima** [comment]
Hey @Zhann 

Could you take a look at the failing test, should be a simple fix:

```
  ×  sous_chef has statement_timeout set to 8mins: PostgreSQL query with errors: SHOW statement_timeout;
     ×  PostgreSQL query with errors: SHOW statement_timeout; output is expected to cmp == /8min/
     
     expected: (?-mix:8min)
          got: 
     psql: FATAL:  database "sous_chef" does not exist
     
     
     (compared using `cmp` matcher)

```

Thanks

File: libraries/helpers.rb (hunk 1 above)
Comment (Zhann) [reply libraries/helpers.rb:131]: I don't think you can (or at least, it's not the smartest idea). You can quote names btw, but then you have to use double quotes, single quotes a syntax error.

I defaulted to the style of other methods in the helper file, assuming that made most sense.

**Comment by Zhann** [comment]
@Xorima fixed, thank you!

**Comment by Zhann** [comment]
16.04 machines seems to fail on `server_install.rb`, but that's not really in scope of (or related to) this change.

**Comment by Zhann** [comment]
Thank you for merging 🙇 

will you be creating a new release for this? (7.1.10)

**Comment by Xorima** [comment]
I will leave for @damacus so he can test the auto release system

**Pull request merged**